src/libraries/Common/src/Interop/Windows/Kernel32/Interop.DeviceIoControl.cs

```diff
@@ -15,6 +15,43 @@ internal static partial class Kernel32
         // https://docs.microsoft.com/windows-hardware/drivers/ddi/ntddstor/ni-ntddstor-ioctl_storage_read_capacity
         internal const int IOCTL_STORAGE_READ_CAPACITY = 0x002D5140;
 
+        // https://learn.microsoft.com/windows/win32/api/winioctl/ni-winioctl-fsctl_set_sparse
+        internal const int FSCTL_SET_SPARSE = 0x000900c4;
+        internal struct FILE_SET_SPARSE_BUFFER
+        {
+            internal int SetSparse;
```

Comment:
BOOLEAN, marshal  per https://learn.microsoft.com/en-us/dotnet/standard/native-interop/best-practices#common-windows-data-types (in general that doc should be solid)